.cargo/config.toml

```diff
@@ -3,7 +3,7 @@ rustflags = ["--cfg", "tokio_unstable"]
 
 [target.x86_64-unknown-linux-gnu]
 linker = "clang"
-rustflags = ["-C", "link-arg=-fuse-ld=lld", "-C", "link-arg=-Wl,--compress-debug-sections=zlib", "--cfg", "tokio_unstable"]
+rustflags = ["-C", "link-arg=-Wl,--compress-debug-sections=zlib,-fuse-ld=lld", "--cfg", "tokio_unstable"]
```

Comment:
After rebasing, I'm getting the following error when building

```
clang-11: error: linker command failed with exit code 137 (use -v to see invocation)
```

Reverting this change allows the build to succeed. I've created an [issue](https://github.com/fedimint/fedimint/issues/3000) (3000, woo!) so we can find a solution that compresses and works with the linker.

Reply:
Might that be a Linux vs. Mac problem again? What's your platform?